src/CoreGraphics/CGFunction.cs

```diff
@@ -103,13 +108,22 @@ protected internal override void Release ()
 		// Apple's documentation says 'float', the header files say 'CGFloat'
 		unsafe delegate void CGFunctionEvaluateCallback (/* void* */ IntPtr info, /* CGFloat* */ nfloat *data, /* CGFloat* */ nfloat *outData); 
 		delegate void CGFunctionReleaseCallback (IntPtr info);
-			
+
+#if NET
+		[StructLayout (LayoutKind.Sequential)]
+		unsafe struct CGFunctionCallbacks {
+			public /* unsigned int */ uint version;
+			public delegate* unmanaged<IntPtr, nfloat*, nfloat*, void> evaluate;
+			public delegate* unmanaged<IntPtr, void> release;
+		}
+#else			
 		[StructLayout (LayoutKind.Sequential)]
 		struct CGFunctionCallbacks {
 			public /* unsigned int */ uint version;
 			public CGFunctionEvaluateCallback? evaluate;
 			public CGFunctionReleaseCallback? release;
 		}
+#endif
```

Comment:
I considered that, but since the struct needs to be marked `unsafe`, this would start to get into the ridiculous range:
```
    [StructLayout (LayoutKind.Sequential)]
#if NET
    unsafe struct CGFunctionCallbacks {
#else
    struct CGFunctionCallbacks {
#endif
        public /* unsigned int */ uint version;
#if NET
        public delegate* unmanaged<IntPtr, nfloat*, nfloat*, void> evaluate;
        public delegate* unmanaged<IntPtr, void> release;
#else
        public CGFunctionEvaluateCallback? evaluate;
        public CGFunctionReleaseCallback? release;
#endif
    }
```

Reply:
It doesn't work if you mark just the fields as unsafe?

```cs
    public unsafe delegate* unmanaged<IntPtr, nfloat*, nfloat*, void> evaluate;
    public unsafe delegate* unmanaged<IntPtr, void> release;
```